### ISSUE TITLE
Visitor complaint about "register" and "contact us"

### ISSUE THREAD

**Issue by marycklein** [WesternFriend/westernfriend.org#1039]
on 2/7/2024, K. Foskett, w-----@msn.com, wrote:

Your website “Register” and “Contact” functions are not working. Please fwd this to you webmaster.

Your website does not let me “register”.  

I completed all required fields and correctly set a password containing four letters (with one uppercase), four numbers, and one punctuation mark;  and I checked the Captcha box, then pressed the button below it to submit.

After 20 - 30 seconds an on-screen message opens saying “Captcha has timed out. Please check the box again.”  This occurs every time I check the Captcha box again.  I’m using an iPad Pro with the most current Apple iOS version (automatically updated.)

(I’m a member of Friends Meeting of Washington, Baltimore Yearly Meeting, for ^4o years, but I originally joined Claremont Friends Meeting ca. 1970).

**Comment by marycklein** [comment]
on 2/8/2024, Debbie Townsend, 2dtownsends@earthlink.net, wrote:

Hi Mary! I've just tried to send an email via the "contact us" screen on the WF website, and received an odd error message in response. Just in case the message didn't come through:

Eastside Friends Meeting would like to order 20 more copies of the North Pacific YM Faith and Practice. Can you please let me know . . . [rest of message deleted here]

**Comment by marycklein** [comment]
Voice message on 2/19/2023, saying "contact us" form would not work. Anna Frances,  (520) 230-0034, who said:

"Oh, yes Mary. My name is Anna and a Francis and I've been trying to I received a gift of subscription to Western friend for Christmas. And I moved so I need to change my address with you all I've tried to do it online. I can fill out the contact form. But when I press submit it doesn't go anywhere. So I don't know what's happening with your website. Anyway, I am mailing a card that has all my contact information . . . "
